lttngpy/src/lttngpy/channel.cpp

```diff
@@ -12,12 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <lttng/lttng-error.h>
```

Comment:
Please add a comment to explain why the headers are not in alphabetical order, with a TODO to remind us to move the include down later once it's fixed:

```suggestion
// Include lttng-error.h before channel.h, otherwise lttng_error_code isn't known
// TODO: order it alphabetically once liblttng-ctl 2.14+ is fixed:
// https://review.lttng.org/c/lttng-tools/+/15165
#include <lttng/lttng-error.h>
```